distribution/docs/src/main/resources/content/_running/directory-monitor-ingest.adoc

```diff
@@ -12,4 +12,5 @@ For more information about configuring a directory to be monitored, see <<_confi
 
 Files placed in the monitored directory will be ingested automatically.
 If a file cannot be ingested, they will be moved to an automatically-created directory named `.errors`.
+More information about the ingest operations can be found in the log located by default in the `/data/log/ingest_error.log` file in the ${branding} directory.
```

Comment:
should this be `${home_directory} instead of `${branding} directory`?

Reply:
Yes, thank you